finalPrep/modZeroHero.js

```diff
@@ -55,30 +87,79 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
-
+var scaryMonster = {
+  name: "Godzilla",
+  smell: "Nuclear",
+  weight: 100000000,
+  citiesDestroyed: ["Tokyo", "New York", "San Francisco"],
+  luckyNumbers: [1, 25, 100],
+  address: {
+    number: 123,
+    street: "Monster Way",
+    state: "NY",
+    zip: 10000
+  }
+}
+console.log(scaryMonster);
 
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
+//   - energyLevel = 50
 
 // - Create the following class methods
 //   - sayName, should print the hero's name to the console
 //   - maximizeEnergy, should update the energyLevel to 1000
 //   - gainPower, should take an argument of a number and INCREASE the powerLevel by that number
 
+class SuperHero{
+  constructor(name, superpower, age, archNemesis, powerLevel, energyLevel){
+    this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+    this.archEnemies = "The Syntax Error";
+    this.powerLevel = 100;
+    this.energyLevel = 50;
+  }
+  sayName(){
+    console.log(this.name);
+  }
+  maximizeEnergy(){
+    this.energyLevel = 1000;
+  }
+  gainPower(newPower){
+    this.powerLevel = this.powerLevel + newPower;
+  }
+}
 // - Create 2 instances of your SuperHero class
-
+var captainArray = new SuperHero("Captain Array", "Control Arrays", 10000);
```

Comment:
It looks like you're only instantiating these `SuperHero`s with the first 3 parameters, so the last 3 are not used (see above comment).